test/system/250-systemd.bats

```diff
@@ -42,7 +42,7 @@ function teardown() {
 
     cname=$(random_string)
     # See #7407 for --pull=always.
-    run_podman create --pull=always --name $cname --label "io.containers.autoupdate=image" $IMAGE top
+    run_podman create --pull=always --name $cname --label "io.containers.autoupdate=local" $IMAGE top
```

Comment:
Please create a new test for the `local` policy.  `--pull=always` and local conflict a bit and the `TODO` not below is a rather important reminder of open work items.